versions.yaml

```diff
@@ -28,6 +30,7 @@ versions:
       - https://istio-release.storage.googleapis.com/charts/cni-1.20.3.tgz
       - https://istio-release.storage.googleapis.com/charts/ztunnel-1.20.3.tgz
   - name: latest
+    version: "1.22"
```

Comment:
`1.22-dev` maybe?

Reply:
No, unfortunately the actual istiod version is `1.22-alpha.44056bc33df10623819256d847848c27066b3279-44056bc33df10623819256d847848c27066b3279-Clean` and obviously changes every time the image is built. But we only update the versions.yaml file once a day. This means that the tests would fail if we used the full version here.

The stable releases report the version as `1.20.3-692e556046b48ebc471205211c68a2c69e74a321-Clean`. The two formats are inconsistent and it's not trivial to extract the important part of the version, so I decided to just use "1.22" as the version here. 

But yeah, I don't like this either. I'll see if I can come up with a regexp that extracts what we want in both cases.

Reply:
Sorry, I meant alpha.